source/state_representation/include/state_representation/State.hpp

```diff
@@ -100,9 +100,10 @@ class State {
   bool is_empty() const;
 
   /**
-   * @brief Setter of the empty attribute to true
+   * @brief Setter of the empty attribute
+   * @param empty bool if the state should be empty or not
    */
-  void set_empty();
+  void set_empty(bool empty = true);
```

Comment:
Please add the default value to the Python binding of `State` as well, since this is a public function